src/components/organisms/ChatDrawer/ChatDrawer.tsx

```diff
@@ -133,6 +145,7 @@ const ChatDrawer: React.FC<PropsType> = ({
       {!isChatDrawerExpanded ? (
         <div className="chat-icon-container">
           <FontAwesomeIcon icon={faCommentDots} className="chat-icon" />
+          {notificationAlert ? (<><span className="chatdot"></span><audio autoPlay ><source src="/sounds/notification.mp3" /></audio></>) : ''}
```

Comment:
When you need to 'render this thing or render nothing', we can use the shorter `&&` pattern, rather than a full 'ternary':

```suggestion
          {notificationAlert && (<><span className="chatdot"></span><audio autoPlay ><source src="/sounds/notification.mp3" /></audio></>)}
```

You can read more about React conditional rendering at:

- https://reactjs.org/docs/conditional-rendering.html

And this `if` pattern is described specifically in:

- https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

We only need to use a 'ternary' for `if`/`else` logic, as described in:

- https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

While it's not directly relevant here, it's also good to know that if you need/want a component to render nothing at all, then you can return `null`, as described in:

- https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

Reply:
While we could use this conditional render logic to selectively render the `<audio>` tag when we need it, it might be simpler to not add any HTML for it at all, but just use the JavaScript audio API directly:

- https://developer.mozilla.org/en-US/docs/Web/API/HTMLAudioElement/Audio

We would call this within a `useEffect` React hook that has the `unreadMessagecount` as one of its dependencies (see above comments for further explanation/documentation on this). That way, the `useEffect` hook will only be run when its dependency changes:

- https://reactjs.org/docs/hooks-reference.html#useeffect

Using the above as resources, it shouldn't be too hard to implement this way. I'll leave the specifics of that up to you, and review it in the next round of code review.

Reply:
We also want to avoid using 'magic strings' to reference images/audio assets/etc. While this is something we currently do a lot of in our codebase, it's one of those 'legacy' patterns that we want to change/eliminate from our code.

Because we are using `create-react-app` (which uses `webpack`), we can more explicitly declare our dependencies by `import`ing files directly within our TypeScript files. See the following docs for more details:

- https://create-react-app.dev/docs/adding-images-fonts-and-files/
  - > You can import a file right in a JavaScript module. This tells webpack to include that file in the bundle. Unlike CSS imports, importing a file gives you a string value. This value is the final path you can reference in your code, e.g. as the src attribute of an image or the href of a link to a PDF.
- https://webpack.js.org/loaders/file-loader/
  - This is used 'under the hood' by create-react-app to support this feature. While we generally don't need to think about it directly, it can be good to know what is used in cases where we need to debug/look deeper into an issue/etc
  - We can see the webpack config that create-react-app uses at the link below, with the highlighted part specifically referring to the 'loaders':
    - https://github.com/facebook/create-react-app/blob/v3.4.1/packages/react-scripts/config/webpack.config.js#L364-L557
  - To import the sound files like this, the final generic 'file loader' is used:
    - https://github.com/facebook/create-react-app/blob/v3.4.1/packages/react-scripts/config/webpack.config.js#L538-L553

Instead of adding the audio to the 'public' directory as you are currently (eg. `<projectRoot>/public/sounds`), create a new directory at `<projectRoot>/src/assets/sounds`, and then add the `.mp3` file there. Then you should be able to import it into this file as follows:

```typescript
import notificationSound from "assets/notification.mp3";
```

And then use that `notificationSound` string as the 'source' for the audio.

---

**Note:** Depending on the IDE you use, and it's inbuilt errors/warnings, it might not be 'smart enough' to know that this is ok. For example, webstorm showed the following error when I used the syntax above, yet I believe the code should work fine:

![image](https://user-images.githubusercontent.com/753891/107594627-8512d500-6c66-11eb-9baa-6c08530c645c.png)

(**note:** I just used a random code file I had open to test the syntax, so don't assume that the file I used in the screenshot is necessarily the file that it should be added to)

If you can run the code, create-react-app doesn't complain, and you can hear the sound when imported this way, then you should be good to go.

Reply:
we use notification from https://www.zedge.net/find/ringtones/notification%20sound "notification tune" from their

Reply:
Thanks. I've just asked @sparkletown/design whether they have any particular sound files in mind for this, you can see the thread here: https://github.com/sparkletown/internal-sparkle-issues/issues/230#issuecomment-777229021